### PR TITLE
R101

### PR DESCRIPTION
**Pull request by BenFradet**

### CODE REVIEW

File: schemas/com.hubspot/deal_deletion/jsonschema/1-0-0 [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      "format": "date-time",
+      "description": "When this event occurred."
+    },
+    "attemptNumber": {
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 2147483647,
+      "description": "Which attempt this is to notify your service of this event (starting at 0). If your service times-out or throws an error as described in 'Retries' below, we will attempt to send the notification to your service again."
+    },
+    "changeFlag": {
```
Comment (BenFradet) [inline schemas/com.hubspot/deal_deletion/jsonschema/1-0-0]: hey @misterpig , what do those `changeFlag`s correspond to? they don't seem to be listed in https://developers.hubspot.com/docs/methods/webhooks/webhooks-overview

File: jsonpaths/com.callrail/call_complete_1.json [hunk 2]
```diff
@@ -1,51 +1,55 @@
...
+        "$.schema.vendor",
+        "$.schema.name",
+        "$.schema.format",
+        "$.schema.version",
+        "$.hierarchy.rootId",
+        "$.hierarchy.rootTstamp",
+        "$.hierarchy.refRoot",
+        "$.hierarchy.refTree",
+        "$.hierarchy.refParent",
+        "$.data.datetime",
```
[the remark this answers is not in the document]
Comment (chuwy) [reply jsonpaths/com.callrail/call_complete_1.json]: I assume it belong to this ticket, which is marked as "blocked" https://github.com/snowplow/iglu-central/issues/534

File: schemas/com.hubspot/deal_deletion/jsonschema/1-0-0 [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+      "minimum": 0,
+      "maximum": 2147483647,
+      "description": "The ID of the object that was created/changed/deleted. For contacts this is the vid; for companies, the companyId; and for deals the dealId."
+    },
+    "changeSource": {
+      "type": "string",
+      "maxLength": 128,
+      "description": "The source of this change. Can be any of the change sources that you find on contact property histories."
+    },
+    "eventId": {
```
Comment (chuwy) [inline schemas/com.hubspot/deal_deletion/jsonschema/1-0-0]: It seems that docs don't mention any of these fields as required. But wondering if it is really the case.

File: jsonpaths/com.callrail/call_complete_1.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (BenFradet) [reply jsonpaths/com.callrail/call_complete_1.json]: so how should we proceed?

[the remark this answers is not in the document]
Comment (chuwy) [reply jsonpaths/com.callrail/call_complete_1.json]: From my point of view we have two options:

1. Just exclude these DDL and JSON Paths from PR. This is what we usually do, its not a big problem as events are still validated against schema, users just don't get these properties populated in their Redshifts.
2. Sumbit `2-0-0` schema with corresponding table.

I'm in strong favor of 1st option.

[the remark this answers is not in the document]
Comment (chuwy) [reply jsonpaths/com.callrail/call_complete_1.json]: Also, if somebody really needs them - they always can upgrade their DB and use custom JSON Path assets.

File: schemas/com.hubspot/deal_deletion/jsonschema/1-0-0 (hunk 1 above)
Comment (misterpig) [reply schemas/com.hubspot/deal_deletion/jsonschema/1-0-0]: Hey @BenFradet , this field pops up for both creation and deletion webhooks on all 3 objects. It's not listed on the reference page but it does show up in all the payloads. Here's a example of a test contact creation payload:

`[{"eventId":1,"subscriptionId":105059,"portalId":5226210,"occurredAt":1551049519100,"subscriptionType":"contact.creation","attemptNumber":0,"objectId":123,"changeSource":"CRM","changeFlag":"NEW","appId":185219}]`

File: schemas/com.hubspot/deal_deletion/jsonschema/1-0-0 (hunk 3 above)
Comment (misterpig) [reply schemas/com.hubspot/deal_deletion/jsonschema/1-0-0]: Hey @chuwy , these fields all appears in actual payloads I've received from the webhooks I've set up on a Hubspot portal. I think it's safe to expect them in any of the payloads.

**Pull request merged**